internal/pkg/manifest/workload_ecs.go

```diff
@@ -91,17 +91,33 @@ type AlarmArgs struct {
 	MemoryUtilization *float64 `yaml:"memory_utilization"`
 }
 
+// WorkerAlarmArgs represents specs of CloudWatch alarms for Worker Service deployment rollbacks.
+type WorkerAlarmArgs struct {
+	AlarmArgs
+	MessagesDelayed *int `yaml:"messages_delayed"`
```

Comment:
This is so strange that ECS recommends this metric 🤔 I'm struggling to understand how this gets used by customers today